src/main/java/com/example/carservice/service/CarServiceImpl.java

```diff
@@ -10,13 +10,23 @@
 public class CarServiceImpl implements CarService {
 
     private final CarRepository repository;
+    private final CarRepository carRepository;
```

Comment:
Why did you inject wice the smae repository?

Reply:
I didn't notice at first that it was already injected. I saw it when I started editing the code after your review.